src/index.html

```diff
@@ -10,7 +10,56 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    class=" search-form"
+    method="get"
+    action="/"
+    data-qa="big"
+    >
+      <label>
+        <div>
+          <img
+          src="images/Search.svg"
+          alt="the icon of a magnifier"
+          class="icon-1"
+          >
```

Comment:
```suggestion
          <img
            src="images/Search.svg"
            alt="the icon of a magnifier"
            class="icon-1"
          >
```
pay attention to the code style